README.md

```diff
@@ -218,7 +218,7 @@ Source-Tree Organization
 ========================
 
 - `libgalois` contains the source code for the shared-memory Galois library, e.g., runtime, graphs, worklists, etc.
-  - Packaged algorithms are include in `libgalois` in two places: `include/galois/analytics` and `src/analytics`.
+  - Packaged algorithms are include in `libgalois` in two places: `include/katana/analytics` and `src/analytics`.
```

Comment:
libkatana?

Reply:
I kind of like keeping the name for the core library `galois`; reserving `katana` as the the name for everything collectively.